tests/ext/sandbox/preinitialized_tracing_of_method.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+[Prehook regression] Trace public static method
+--ENV--
+_DD_LOAD_TEST_INTEGRATIONS=1
+--FILE--
+<?php
+use DDTrace\SpanData;
+
+class Test
+{
+    public function automaticaly_traced_method()
+    {
+        echo "AUTOMATICALY_TRACED_METHOD" . PHP_EOL;
+    }
+}
+
+function tracing_function(SpanData $span, array $args) {
```

Comment:
@labbati 
After some cleanup - I decided to give this approach a go, and with tiny changes its now working.